lib/active_admin/application.rb

```diff
@@ -271,10 +271,12 @@ def controllers_for_filters
   private
 
     def register_default_assets
-      register_stylesheet 'active_admin.css',       media: 'screen'
-      register_stylesheet 'active_admin/print.css', media: 'print'
+      ActiveSupport::Deprecation.silence do
```

Comment:
This was the easiest solution but we can skip it. We'd have to delete the code here though and add these as defaults in AssetRegistration. Do you prefer that?